.github/workflows/ci.yml

```diff
@@ -20,6 +20,8 @@ jobs:
           - 'nightly'
         os:
           - ubuntu-latest
+          - macOS-latest
+          - windows-latest
```

Comment:
```suggestion
          include:
          - os: windows-latest
            version: '1'
            arch: x86
```

Reply:
In other repos, I've just been including a single windows 32-bit job; this minimizes the amount of CI time we're spending (doing our little part to not stress the system too much), while also providing a good mix of arch/os testing; mac tends to be very similar to linux testing wise, so windows is really the only other platform it's good to sanity check.

Reply:
Done!